app/scripts/controllers/cluster.js

```diff
@@ -18,10 +18,16 @@ angular.module('cluster', ['stats', 'sql', 'common', 'nodeinfo'])
 
         var firstRedirect = false;
 
+        $scope.goToPath = function (id) {
```

Comment:
is this function also used in the HTML templates? if not, it does not be on $scope
you could call $state.go('nodes.node', {}) directly on line 30 then